nginx_proxy/config.yaml

```diff
@@ -26,6 +26,9 @@ options:
     active: false
     default: nginx_proxy_default*.conf
     servers: nginx_proxy/*.conf
+  proxy_protocol:
+    enabled: false
+    real_ip_from: null
```

Comment:
Since `real_ip_from` is optional, there must be no default value, otherwise it comes mandatory again. With this current config, the user is requested to set a `real_ip_from` even when `enabled: false` is set :see_no_evil: 

```suggestion
```